### PR TITLE
Limbus Company E.G.O dump

### PR DESCRIPTION
**Pull request by vampirebat74**
## About The Pull Request

Adds E.G.O for upcoming abnormalities from Limbus Company, particularly those from Canto IV.

Untitled

Farmwatch sprites by **Mel (meltaculo)**, Sloshing sprites by **Choccymilkshake enjoyer#6306**, AEDD suit by **Miller (bluhnt)**, spicebush sprites and animations by **Ubage (ubage)**

## Why It's Good For The Game

More loot for the wishing well, saves time for upcoming content and makes them more atomic.

## Changelog
:cl:
add: Adds roseate desire, sloshing, red sheet, capote, AEDD, farmwatch and spicebush
soundadd: sound effects for pink shoes, farmwatch and spicebush
imageadd: adds relevant E.G.O icons
/:cl:

### CODE REVIEW

**Review by EgorDinamit — COMMENTED** [review]
Is this PR ready?

**Review by Kitsunemitsu — APPROVED** [review]
Standard EGO gear update.

**Comment by vampirebat74** [comment]
> Is this PR ready?

No, spicebush isn't finished yet. Converting this PR into a draft until then

File: code/game/objects/items/ego_weapons/he.dm [hunk 1]
```diff
@@ -1346,3 +1346,71 @@
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 40
 							)
+
+/obj/item/ego_weapon/roseate
+	name = "roseate desire"
+	desc = "A salacious cat o' nine tails made from pink ribbons. Merely touching it ushers a primal excitement."
+	special = "The damage of this weapon is significantly lowered depending on the user's temperance attribute."
+	icon_state = "roseate_desire"
+	force = 35
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/he.dm:1355]: It's passive actively LOWERS it's damage, with the minimum TYPICALLY being a 20% decrease, to 28 force on a slower than normal weapon. It needs MORE force. Make it WAW-like if you want, the fact it has inverse scaling with temperance nerfs the hell out of it.

File: code/game/objects/items/ego_weapons/he.dm [hunk 2]
```diff
@@ -1346,3 +1346,71 @@
...
+		charged = FALSE
+
+/obj/item/ego_weapon/aedd/proc/power_attack(mob/living/target, mob/living/user)
+	var/userjust = (get_attribute_level(user, JUSTICE_ATTRIBUTE))
+	var/justicemod = 1 + userjust/100
+	target.apply_damage((force *= justicemod), BLACK_DAMAGE, null, target.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
+	playsound(src, 'sound/abnormalities/thunderbird/tbird_charge.ogg', 50, TRUE)
+	var/turf/T = get_turf(target)
+	new /obj/effect/temp_visual/justitia_effect(T)
+	force = initial(force)
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/he.dm:1416]:
```suggestion
	target.apply_damage((force * justicemod), BLACK_DAMAGE, null, target.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
	playsound(src, 'sound/abnormalities/thunderbird/tbird_charge.ogg', 50, TRUE)
	var/turf/T = get_turf(target)
	new /obj/effect/temp_visual/justitia_effect(T)
```

File: code/game/objects/items/ego_weapons/teth.dm [hunk 3]
```diff
@@ -428,3 +428,60 @@
...
+			var/mob/living/simple_animal/M = target
+			if(!ishuman(M) && !M.has_status_effect(/datum/status_effect/rend_black))
+				to_chat(user, "A talisman from [src] sticks onto [target]!")
+				new /obj/effect/temp_visual/talisman(get_turf(M))
+				M.apply_status_effect(/datum/status_effect/rend_black)
+				hit_count = 0
+
+/obj/item/ego_weapon/shield/capote
+	name = "capote"
+	desc = "Charge me with all your strength! Your horns cannot pierce my soul!"
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/teth.dm:471]: Very good description!

File: code/game/objects/items/ego_weapons/teth.dm [hunk 4]
```diff
@@ -428,3 +428,60 @@
...
+	worn_icon = 'icons/obj/clothing/belt_overlays.dmi'
+	worn_icon_state = "capote"
+	force = 22
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("pokes", "jabs", "tears", "lacerates", "gores")
+	attack_verb_simple = list("poke", "jab", "tear", "lacerate", "gore")
+	hitsound = 'sound/weapons/ego/spear1.ogg'
+	reductions = list(20, 20, 20, 0)
+	block_duration = 0.5 SECONDS
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/teth.dm:482]:
```suggestion
	block_duration = 1 SECONDS
```

File: code/modules/spells/ability_types/realized.dm [hunk 5]
```diff
@@ -436,3 +436,78 @@
...
+	. = ..()
+	QDEL_IN(src, 15 SECONDS)
+
+/mob/living/simple_animal/hostile/farmwatch_plant/death()
+	density = FALSE
+	animate(src, alpha = 0, time = 1)
+	QDEL_IN(src, 1)
+	..()
+
+/mob/living/simple_animal/hostile/spicebush_plant
```
Comment (LanceSmites328) [inline code/modules/spells/ability_types/realized.dm:467]: I believe if these don't have their faction set to `list("neutral")` they'll be ignored by most enemies.

File: code/modules/spells/ability_types/realized.dm [hunk 6]
```diff
@@ -436,3 +436,78 @@
...
+
+/mob/living/simple_animal/hostile/spicebush_plant/proc/HealPulse()
+	pulse_cooldown = world.time + pulse_cooldown_time
+	//playsound(src, 'sound/abnormalities/rudolta/throw.ogg', 50, FALSE, 4)//TODO: proper SFX goes here
+	for(var/mob/living/L in livinginview(8, src))
+		if(faction_check_mob(L))
+			continue
+		L.adjustBruteLoss(-2)
+		var/mob/living/carbon/human/H = L
+		H.adjustSanityLoss(-2)
```
Comment (LanceSmites328) [inline code/modules/spells/ability_types/realized.dm:513]:
```suggestion
	for(var/mob/living/L in livinginview(8, src))
		if(faction_check_mob(L))
			continue
		L.adjustBruteLoss(pulse_damage)
		if(ishuman(L))
			continue
		var/mob/living/carbon/human/H = L
		H.adjustSanityLoss(pulse_damage)
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/spells/ability_types/realized.dm (hunk 5 above)
Comment (vampirebat74) [reply code/modules/spells/ability_types/realized.dm:467]: done, for both of the summons.

**Review by Kitsunemitsu — APPROVED** [review]
All issues resolved, Ready to go.

**Review by Kitsunemitsu — APPROVED** [review]
One was missed, reapproving.

**Pull request merged**